composer.json

```diff
@@ -10,8 +10,8 @@
 	"require": {
 		"silverstripe/framework": "^4.0"
 	},
-	"extra": {
-		"installer-name": "siteconfig"
```

Comment:
`extra` was duplicated in composer.json and was being ignored. Look down the file a bit...

Also it's not needed. `silverstripe/siteconfig` is already named as `siteconfig` and doesn't need an alias.